dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SourceFileHelper.java

```diff
@@ -29,12 +33,63 @@
 
 public final class SourceFileHelper {
 
+    private static final long BYTES_PER_MEGABYTE = 1024L * 1024L;
+    private static final long MAXIMUM_FILE_SIZE = BYTES_PER_MEGABYTE;
+    private static final long NOTEBOOK_MAXIMUM_FILE_SIZE = 3 * BYTES_PER_MEGABYTE;
```

Comment:
Probably overkill, but you could consider making this a configurable setting, so if someday a user really wants to register a 3.5MB notebook and we want to allow it, we could allow it without having to create new build.